### PR TITLE
Typos in S2

### PR DESCRIPTION
**Pull request by ammedmar**

### CODE REVIEW

**Review by devsinha — COMMENTED** [review]
I am leaving a comment.

File: parts/s6.tex [hunk 1]
```diff
@@ -442,7 +442,7 @@ \subsubsection{Dualization in cubes}\label{S: dual cubes}
 \begin{itemize}
 \item If $i\in F_{01}$ (i.e.\ the coordinate $x_i$ is free in $F$), then $x_i=1/2$ in $F^\vee$.
 
-\item If $i\in F_0$ (i.e\ the coordinate $x_i$ is bound to $0$ in $F$), then $x_i$ is free in $[0,1/2]$ in $F^\vee$.
+\item If $i\in F_0$ (i.e.\ the coordinate $x_i$ is bound to $0$ in $F$), then $x_i$ is free in $[0,1/2]$ in $F^\vee$.
```
Comment (devsinha) [inline parts/s6.tex:445]: Here is a comment associated to line 445.

**Comment by gf2078** [comment]
That didn't work. How do I make this thing go away?

**Comment by gf2078** [comment]
What happened to my last comment?

**Comment by gf2078** [comment]
Nevermind - somehow moved on to a different pull request. 
Still not sure how to dismiss them, though

**Review by gf2078 — APPROVED** [review]
Look's good

**Pull request merged**